sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/aad/implementation/AzureActiveDirectoryConfiguration.java

```diff
@@ -41,7 +41,8 @@
     public AzureClientRegistrationRepository clientRegistrationRepository() {
         return new AzureClientRegistrationRepository(
             createDefaultClient(),
-            createAuthzClients());
+            createAuthzClients(),
+            properties.getAuthorization());
```

Comment:
Can we pass `properties` instead of `properties.getAuthorization()`? Maybe we need other property item in `AzureClientRegistrationRepository`.